### ISSUE TITLE
[Improvement] Support Tailwind arbitrary borders

### ISSUE THREAD

**Issue by davestewart** [bernaferrari/FigmaToCode#118]
AFAIKT it looks like arbitrary border sizes are not supported in Tailwind.

I'm adding support in the Tailwind renderer, so would be a nice add at some point.

**Comment by bernaferrari** [comment]
Good catch. And probably rings. I guess border outside is a ring?

**Comment by davestewart** [comment]
I'm not too sure of all the Tailwind ins and outs TBH!

**Comment by bernaferrari** [comment]
yeah, I mean, I don't remember if we have ring or not, but if you want outside stroke to be a ring, I can add (or you can do too)

**Comment by davestewart** [comment]
How about I get the Tailwind renderer over the line, then we'll be in a better position to see what works

**Comment by bernaferrari** [comment]
Yeah, go for it

**Comment by bernaferrari** [comment]
I'm supporting rings now in https://github.com/bernaferrari/FigmaToCode/pull/184

**Comment by bernaferrari** [comment]
if node.strokeAlign is center or outside => ring
if none doesn't have uniform borders => inside (no mater stroke align, but if strokealign is center or outside, will emit a warning to the user).
